webapp/app/forms/steps/lotse/steuerminderungen.py

```diff
@@ -68,7 +68,7 @@ def render(self, **kwargs):
                                component='StmindSelectionPage',
                                props=props_dict,
                                # TODO: These are still required by base.html to set the page title.
-                               form=self.form,
+                               form=self.render_info.form,
```

Comment:
How could this not be verified by a test?

Reply:
Good question. We do not have a test to check what parameters we set for the Jinja template and the form is only used to decide in the `base.html` whether to show "Fehler" in the title. The `props_dict` is correctly filled because we use the form object for the creation of `props_dict`.

We could think about adding tests for this